EIPS/eip-1.md

```diff
@@ -31,59 +31,49 @@ There are three types of EIP:
 
 ## EIP Work Flow
 
-The EIP repository editors change the EIPs status. Please send all EIP-related email to the EIP editors or preferably (e.g. for public transparency) you can ping one of the Editors that is involved with your pull request (PR); they are listed under [EIP editors below](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-editors). Also see [EIP Editor Responsibilities & Workflow](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-editor-responsibilities-and-workflow).
+The normal process is:
+
+```
+[ A pull request ] -> [ DRAFT ] -> [ LAST CALL ] -> [APPROVED] -> [ FINAL ]
```

Comment:
I'm still concerned this oversimplifies matters, especially since this only describes the process for non-core EIPs - the process for core EIPs is different.

Also, 'approved' isn't part of the non-core process; it goes straight from last call to final.

It's more like:

Core EIPs:

 Draft -> Editor last call -> Approved(?) -> Accepted -> Final

Non-core EIPs:

 Draft -> Last call -> Final

Reply:
My understanding is that Last Call will cause a core EIP to go from this EIP process to the core dev meeting, where upon the best outcome is that it is approved. Then at some point it is deployed to a majority of clients.

The fact that EF deploys it to EF-supported clients seems immaterial from our EIP perspective. The time when 51% of clients are running it seems relevant from our perspective.

I am not trying to change things here or rock the boat. I am honestly trying to get Last Call in, which of course touches into these other processes.

Please help me understand how to fix this and get it done. Also, please give me a call so we can figure this out together.

---

I am not proposing this now, but another option is:

```
Non-core: [ DRAFT ] -> [ LAST CALL ] -> [ FINAL ]
Core:     [ DRAFT ] -> [ LAST CALL ] -> [ FINAL ] -> [ DEPLOYED ]
```

Where non-core has EIP editors making "final" decision. And Core has Core Devs making "final" decision. Then "deployed" is merely a review of 51% deployment (adjudicated by EIP editors). This is how I understand the process as actually happening.

Reply:
> My understanding is that Last Call will cause a core EIP to go from this EIP process to the core dev meeting, where upon the best outcome is that it is approved. Then at some point it is deployed to a majority of clients.

The process for Core EIPs has always involved back-and-forward between the EIP process and core devs. Progressing to the 'Accepted' and 'Final' states require approval from All Core Devs, with 'Final' requiring that it's been implemented for a future hard fork.

I don't see anything in this PR that changes that, so it's important that any proposed diagrams adequately describe the actual process.

Reply:
I am using "accepted as draft", clearly that wording doesn't make sense if "accepted" is a status too.

I thought the core devs only made the decision to implement. I thought the users decided to upgrade and the EIP editors to recognize when that upgrade was deployed.

It is not clear to me how to update my wording to fix it and get this accepted. It is also not clear to me how the process actually works or how it is written on paper.

This is why I asked for a phone call. I am documenting these things and I can make it great I just need to understand it first for myself.

Reply:
One issue is your note "closing PRs as soon as possible by either merging or rejecting them" -- this does not match the diagram.

I want to understand this fully but all the questions and answers here are piecemeal. If I understand it completely I can write it well.

Reply:
> I am using "accepted as draft", clearly that wording doesn't make sense if "accepted" is a status too.

The status you want is just 'Draft', not 'Accepted as Draft'.

> I thought the core devs only made the decision to implement. I thought the users decided to upgrade and the EIP editors to recognize when that upgrade was deployed.

For Core EIPs, the devs decide to implement, and when to configure a hard fork including the changes. If that hard fork is successful, then the state changes to Final.

> It is not clear to me how to update my wording to fix it and get this accepted. It is also not clear to me how the process actually works or how it is written on paper.

I already provided a suggestion for an alternate wording; what was wrong with that?

> One issue is your note "closing PRs as soon as possible by either merging or rejecting them" -- this does not match the diagram.

The diagram shows the states that EIPs can be in, not the process for updating them. A PR is how you update the state or contents of an EIP.